openpype/hosts/max/plugins/publish/validate_frame_range.py

```diff
@@ -27,38 +28,58 @@ class ValidateFrameRange(pyblish.api.InstancePlugin,
 
     label = "Validate Frame Range"
     order = ValidateContentsOrder
-    families = ["maxrender"]
+    families = ["camera", "maxrender",
+                "pointcache", "pointcloud",
+                "review", "redshiftproxy"]
     hosts = ["max"]
     optional = True
     actions = [RepairAction]
 
     def process(self, instance):
         if not self.is_active(instance.data):
-            self.log.info("Skipping validation...")
+            self.log.debug("Skipping Validate Frame Range...")
             return
-        context = instance.context
 
-        frame_start = int(context.data.get("frameStart"))
-        frame_end = int(context.data.get("frameEnd"))
-
-        inst_frame_start = int(instance.data.get("frameStart"))
-        inst_frame_end = int(instance.data.get("frameEnd"))
+        frame_range = get_frame_range(
+            asset_doc=instance.data["assetEntity"])
 
+        inst_frame_start = instance.data.get("frameStart")
+        inst_frame_end = instance.data.get("frameEnd")
+        frame_start_handle = frame_range["frameStart"] - int(
+            frame_range["handleStart"]
+        )
+        frame_end_handle = frame_range["frameEnd"] + int(
+            frame_range["handleEnd"]
+        )
         errors = []
-        if frame_start != inst_frame_start:
+        if frame_start_handle != inst_frame_start:
             errors.append(
                 f"Start frame ({inst_frame_start}) on instance does not match " # noqa
-                f"with the start frame ({frame_start}) set on the asset data. ")    # noqa
-        if frame_end != inst_frame_end:
+                f"with the start frame ({frame_start_handle}) set on the asset data. ")    # noqa
+        if frame_end_handle != inst_frame_end:
             errors.append(
                 f"End frame ({inst_frame_end}) on instance does not match "
-                f"with the end frame ({frame_start}) from the asset data. ")
+                f"with the end frame ({frame_end_handle}) "
+                "from the asset data. ")
 
         if errors:
             errors.append("You can use repair action to fix it.")
-            raise PublishValidationError("\n".join(errors))
+            report = "Frame range settings are incorrect.\n\n"
+            for error in errors:
+                report += "- {}\n\n".format(error)
```

Comment:
How about avoiding the **You can use repair action to fix it** to also be a bullet point?

```suggestion
            bullet_point_errors = "\n".join("- {}".format(error) for error in errors)
            report = (
                "Frame range settings are incorrect.\n\n"
                f"{bullet_point_errors}\n\n"
                "You can use repair action to fix it."
            )
```